bindings/python/CMakeLists.txt

```diff
@@ -41,9 +41,7 @@ function(make_bindings)
     ${PYLIB_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   )
 
-  target_link_libraries(
-    ${PYLIB_NAME} PUBLIC ${PROJECT_NAME} eigenpy::eigenpy proxsuite-nlp::pyproxsuite_nlp
-  )
+  target_link_libraries(${PYLIB_NAME} PUBLIC ${PROJECT_NAME} eigenpy::eigenpy)
```

Comment:
The python bindings depends of some code in the proxsuite-nlp python binding (the polymorphic support by exemple).
So you need to keep the link. There is no problem when proxsuite-nlp is installed but the missing link is a problem when using proxsuite-nlp in a workspace.

Reply:
> The python bindings depends of some code in the proxsuite-nlp python binding (the polymorphic support by exemple). So you need to keep the link. There is no problem when proxsuite-nlp is installed but the missing link is a problem when using proxsuite-nlp in a workspace.

That might be a problem since pyproxsuite-nlp is now a MODULE library

Reply:
We can do this

```cmake
target_include_directories(${PYLIB_NAME}
    PUBLIC
    "$<TARGET_PROPERTY:proxsuite-nlp::pyproxsuite_nlp,INCLUDE_DIRECTORIES>"
)
```

Reply:
Didn't know that !
Very smart